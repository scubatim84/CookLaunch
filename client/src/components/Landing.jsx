import React from 'react';
import RegisterForm from './Auth/RegisterForm';
import {useStylesLanding} from '../Styles';
import {Grid} from '@material-ui/core';

function Landing(props) {
  const classes = useStylesLanding();

  return (
    <div className='landing-background background-image-full'>
      <div className={classes.root}>
        <Grid className={classes.position} item xs={4}>
          <RegisterForm
            handleLoggedIn={props.handleLoggedIn}
            isLoggedIn={props.isLoggedIn}
          />
        </Grid>
      </div>
    </div>
  );
}

export default Landing;
