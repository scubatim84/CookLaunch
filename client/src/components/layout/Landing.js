import React from "react";

import Navbar from "./Navbar";
import LandingBody from "./LandingBody";

function Landing() {
	return (
    <div>
      <Navbar />
      <LandingBody />
    </div>
	);
}

export default Landing;