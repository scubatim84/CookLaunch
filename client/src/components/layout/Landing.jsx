import React from 'react';
import RegisterForm from '../auth/RegisterForm';

import {Grid} from '@material-ui/core';
import {makeStyles} from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
  root: {
    flexGrow: 1,
  },
  position: {
    padding: '125px 0px 100px 20px',
    marginLeft: '0px',
  },
}));

function Landing(props) {
  const classes = useStyles();

  return (
    <div className='landing-background background-image-full'>
      <div className={classes.root}>
        <Grid className={classes.position} item xs={4}>
          <RegisterForm
            handleLoggedIn={props.handleLoggedIn}
            isLoggedIn={props.isLoggedIn}
          />
        </Grid>
      </div>
    </div>
  );
}

export default Landing;
