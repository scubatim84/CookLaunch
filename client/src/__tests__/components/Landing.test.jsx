import React from 'react';
import { render } from '@testing-library/react';

import Landing from '../../components/Landing';

describe('Landing', () => {
  it('Renders component without crashing', () => {
    render(<Landing />);
  });
});
