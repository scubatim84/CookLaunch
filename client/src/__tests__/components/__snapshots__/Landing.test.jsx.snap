import React from 'react';
import {shallow} from 'enzyme';
import Landing from '../../components/Landing';

describe('Landing', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<Landing />);

    expect(wrapper).toMatchSnapshot();
  });
});
